src/commonTest/kotlin/org/petitparser/grammar/lisp/Lisp.kt

```diff
@@ -0,0 +1,71 @@
+package org.petitparser.grammar.lisp
+
+import org.petitparser.core.grammar.Grammar
+import org.petitparser.core.parser.Parser
+import org.petitparser.core.parser.combinator.*
+import org.petitparser.core.parser.consumer.CharPredicate.Companion.any
+import org.petitparser.core.parser.consumer.CharPredicate.Companion.anyOf
+import org.petitparser.core.parser.consumer.CharPredicate.Companion.char
+import org.petitparser.core.parser.consumer.CharPredicate.Companion.pattern
```

Comment:
These are predicates, you want the parsers:

```kotlin
import org.petitparser.core.parser.consumer.any
import org.petitparser.core.parser.consumer.anyOf
import org.petitparser.core.parser.consumer.char
import org.petitparser.core.parser.consumer.digit
```

Possibly PetitParser could reduce their visibility?

Reply:
Maybe prefixing the `predicates` with `is`, i.e. (`isAny, isChar`) will help differentiate the two sets?